extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -0,0 +1,422 @@
+			value: constants.MIGRATION_READINESS_LABEL,
+			height: 16,
+			CSSStyles: {
+				...styles.TOOLBAR_CSS
+			},
+		}).component();
+
+		const readinessComponentContainer = view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'display': 'flex',
+				'flex-direction': 'row',
+				'gap': '8px'
+			}
+		}).component();
+
+		const ready = this.createResultComponentContainer(view, AssessmentResultType.READY);
+		const neadsReview = this.createResultComponentContainer(view, AssessmentResultType.NEEDS_REVIEW);
+		const notReady = this.createResultComponentContainer(view, AssessmentResultType.NOT_READY);
+
+		readinessComponentContainer.addItem(ready);
+		readinessComponentContainer.addItem(neadsReview);
+		readinessComponentContainer.addItem(notReady);
+
+		container.addItem(migrationReadinessText);
+		container.addItem(readinessComponentContainer);
+		return container;
+	}
+
+	private createAssessmentFindingsContainer(view: azdata.ModelView) {
+		const container = view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'display': 'flex',
+				'flex-direction': 'column',
+				'gap': '4px'
+			}
+		}).component();
+
+		const assessmentFindingsText = view.modelBuilder.text().withProps({
+			value: constants.ASSESSMENT_FINDINGS_LABEL,
+			height: 16,
+			CSSStyles: {
+				...styles.TOOLBAR_CSS
+			},
+		}).component();
+
+		const assessmentFindingsComponentContainer = view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'display': 'flex',
+				'flex-direction': 'row',
+				'gap': '8px'
+			}
+		}).component();
+
+		const blockers = this.createResultComponentContainer(view, AssessmentResultType.BLOCKERS,);
+		const warnings = this.createResultComponentContainer(view, AssessmentResultType.WARNINGS);
+
+		assessmentFindingsComponentContainer.addItem(blockers);
+		assessmentFindingsComponentContainer.addItem(warnings);
+
+		container.addItem(assessmentFindingsText);
+		container.addItem(assessmentFindingsComponentContainer);
+		return container;
+	}
+
+	private createResultComponentContainer(view: azdata.ModelView, assessmentResultType: AssessmentResultType) {
+		const container = view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'display': 'flex',
+				'flex-direction': 'column',
+			}
+		}).component();
+
+		const label = view.modelBuilder.text().withProps({
+			value: "",
+			height: 16,
+			CSSStyles: {
+				...styles.TOOLBAR_CSS
+			},
+		}).component();
+
+		const value = view.modelBuilder.text().withProps({
+			value: "",
+			height: 22,
+		}).component();
+
+
+		switch (assessmentResultType) {
+			case AssessmentResultType.READY:
+				label.value = constants.READY;
+				value.CSSStyles = {
+					...styles.ASSESSMENT_SUMMARY_CARD_CSS,
+					'color': '#57A300',
+				};
+				this._readyText = value;
+				break;
+			case AssessmentResultType.NEEDS_REVIEW:
+				label.value = constants.NEEDS_REVIEW;
+				value.CSSStyles = {
+					...styles.ASSESSMENT_SUMMARY_CARD_CSS,
+					'color': '#DB7500',
```

Comment:
colors are defines in helper.ts file, you can use from there or define your new color code there